src/pages/Environments.js

```diff
@@ -108,6 +150,12 @@ const Environments = () => {
     if (!_.some({ id: getDeleteDiskId() }, newDisks)) {
       setDeleteDiskId(undefined)
     }
+    if (!_.some({ appName: getErrorAppId() }, newApps)) {
```

Comment:
These getters surprise me. Can you explain the need for this (offline if needed) over using the raw form?

Reply:
This appears to be the pattern that's been used since the earlier days of Terra UI. I see [Brett's comment](https://github.com/DataBiosphere/terra-ui/blob/dbf333204e2c10aa36cdefce08765ca77c0890af/src/libs/utils.js#L335) noting a use case. If it isn't necessary, I think it'd be good to drop all cases for consistency, which should be beyond the scope of this PR.